src/SWP/Bundle/CoreBundle/Model/FacebookInstantArticlesArticle.php

```diff
@@ -0,0 +1,144 @@
+<?php
+
+/*
+ * This file is part of the Superdesk Web Publisher Core Bundle.
+ *
+ * Copyright 2016 Sourcefabric z.ú. and contributors.
+ *
+ * For the full copyright and license information, please see the
+ * AUTHORS and LICENSE files distributed with this source code.
+ *
+ * @copyright 2016 Sourcefabric z.ú
+ * @license http://www.superdesk.org/license
+ */
+
+namespace SWP\Bundle\CoreBundle\Model;
+
+use SWP\Bundle\ContentBundle\Model\ArticleInterface;
+use SWP\Component\Common\Model\TimestampableTrait;
+use SWP\Component\MultiTenancy\Model\TenantAwareInterface;
+use SWP\Component\MultiTenancy\Model\TenantAwareTrait;
+use SWP\Component\Storage\Model\PersistableInterface;
+
+class FacebookInstantArticlesArticle implements FacebookInstantArticlesArticleInterface, PersistableInterface, TenantAwareInterface
```

Comment:
this class should be placed in FacebookInstantArticlesBundle. Here this class should just contain `protected $article;` property and `TenantAwareTrait` as this is what integrates two bundles. The same way you did with `Application` class